Makefile

```
@@ -126,20 +126,20 @@ build_batcher_client:
 batcher/client/target/release/batcher-client:
 	@cd batcher/client && cargo b --release
 
-batcher_send_sp1_task: batcher/client/target/release/batcher-client
+batcher_send_sp1_task:
 	@echo "Sending SP1 fibonacci task to Batcher..."
-	@cd batcher/client/target/release && ./batcher-client \
+	@cd batcher/client/ && cargo run --release -- \
 		--proving_system SP1 \
-		--proof ../../test_files/sp1/sp1_fibonacci.proof \
-		--vm_program ../../test_files/sp1/sp1_fibonacci-elf
+		--proof test_files/sp1/sp1_fibonacci.proof \
+		--vm_program test_files/sp1/sp1_fibonacci-elf
 
 batcher_send_groth16_task: batcher/client/target/release/batcher-client
```

remove requirement `batcher/client/target/release/batcher-client` if using cargo run
```suggestion
batcher_send_groth16_task:
```